spec/integration/ClientSpec.php

```diff
@@ -287,4 +286,129 @@ function it_receives_the_expected_response_when_looking_up_all_notifications() {
 
     }
 
+    function it_receives_the_expected_response_when_looking_up_a_template() {
+      $templateId = getenv('SMS_TEMPLATE_ID');
+
+      // Retrieve sms notification by id and verify contents
+      $response = $this->getTemplate( $templateId );
+
+      //
+      $response->shouldBeArray();
+      $response->shouldHaveKey( 'id' );
+      $response->shouldHaveKey( 'type' );
+      $response->shouldHaveKey( 'created_at' );
+      $response->shouldHaveKey( 'updated_at' );
+      $response->shouldHaveKey( 'created_by' );
+      $response->shouldHaveKey( 'version' );
+      $response->shouldHaveKey( 'body' );
+      $response->shouldHaveKey( 'subject' );
+
+      $response['id']->shouldBeString();
+      $response['id']->shouldBe( $templateId );
+      $response['type']->shouldBeString();
+      $response['type']->shouldBe( 'sms' );
+      $response['version']->shouldBeInteger();
+      $response['body']->shouldBe("Hello ((name))\n\nFunctional Tests make our world a better place");
+      $response['subject']->shouldBeNull();
```

Comment:
Do we also want to add another test like this for email so that ensure subject has the right value?